assignments/closure.js

```diff
@@ -1,11 +1,19 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
 
+const fun = () => {
+  return ('this is an example of function scope aka local scope');
+}
+
 
 // ==== Challenge 2: Create a counter function ====
+let enumb = 0;
 const counter = () => {
+
```

Comment:
Thomas, this counter does count, but it doesn't return a counter variable. You've got to use a closure here. 
Here is an example:
```
function counter() {
  var x = 0;
  // Closure here
  function newCounter() {
    return x = x + 1;
  }
  newCounter();
  return newCounter;
}

var number = counter();

console.log(number());
console.log(number());
```

Please let me know if you have any questions on this!